src/utils/spo.ts

```diff
@@ -1485,6 +1488,122 @@ export const spo = {
     await request.post(requestOptions);
   },
 
+  async getFileAsListItemByUrl(webUrl: string, url: string, logger?: Logger, verbose?: boolean): Promise<any> {
```

Comment:
shouldn't we add a description comment for this method as we have for the other methods?

Reply:
We definitely should as it's a function that we export which makes it a part of our internal API